services/wallet/community/manager.go

```diff
@@ -0,0 +1,73 @@
+package community
+
+import (
+	"database/sql"
+	"fmt"
+	"time"
+
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/status-im/status-go/services/wallet/thirdparty"
+)
+
+const failedCommunityFetchRetryDelay = 1 * time.Hour
+
+type Manager struct {
+	db *DataDB
+}
+
+func NewManager(db *sql.DB) *Manager {
+	return &Manager{
+		db: NewDataDB(db),
+	}
+}
+
+func (cm *Manager) SetCommunityInfo(id string, c *thirdparty.CommunityInfo) (err error) {
```

Comment:
this probably shouldn't be called from outside this class, better make it private